src/test/java/org/kcctl/command/GetOffsetsCommandTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ *  Copyright 2021 The original authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.kcctl.command;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.kcctl.IntegrationTest;
+import org.kcctl.IntegrationTestProfile;
+import org.kcctl.service.ConnectorOffset;
+import org.kcctl.service.ConnectorOffsets;
+import org.kcctl.support.InjectCommandContext;
+import org.kcctl.support.KcctlCommandContext;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.debezium.testing.testcontainers.DebeziumContainer;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import picocli.CommandLine;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+@QuarkusTest
+@TestProfile(IntegrationTestProfile.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class GetOffsetsCommandTest extends IntegrationTest {
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @InjectCommandContext
+    KcctlCommandContext<GetOffsetsCommand> context;
+
+    @BeforeAll
+    public static void prepare() {
+        // TODO: replace explicit container tag with DebeziumContainer.latestStable() once Debezium releases 2.4, which is based on Kafka 3.5
```

Comment:
Could you log a follow-up issue so we don't forget about it?

Reply:
👍 Done: https://github.com/kcctl/kcctl/issues/346